docs/user-guide/configuring.md

```diff
@@ -96,6 +96,7 @@ An environment defines global variables that are predefined. The available envir
 
 * `browser` - browser global variables.
 * `node` - Node.js global variables and Node.js scoping.
+* `commonjs` - CommonJS global variables and CommonJS scoping (use this for browser-only code that uses Browserify/WebPack).
```

Comment:
I think you're delving a lot deeper into philosophy here than most of our end-users will. I don't this to be overly prescriptive or restrictive, just to point out that it's different from `node`.